best-practices/project-standards.rst

```diff
@@ -0,0 +1,51 @@
+Project Standards
+=================
+
+Release Engineering strives to use consistent tools and style conventions
+across all projects. To that end, the `Release Engineering Project Standard`_
+(or REPS) was created.
+
+This repo contains both the REPS definition, as well as a tool that can be used
+to bootstrap new projects that conform to this definition.
+
+Standard Definition
+-------------------
+
+The standards themselves are defined in the `STANDARD.md`_ file at the root of
+the repo.
+
+Bootstrapping new Projects
+--------------------------
+
+The repo also provides a bootstrapping tool called ``reps-new``. The
+recommended way to run it is with `pipx`_:
+
+.. code-block:: bash
+
+   $ pipx run reps-new
+
+Updating the Standards
+----------------------
+
+These project standards are a living document, and it's encouraged that new
+tools, workflows and modifications be added as our best practices evolve.
+
+If you would like to adjust a standard, the recommended process is to:
+
+1. `Create an issue`_ describing the change you'd like to make, share it with
+   the team and get general consensus that it is worth making.
+2. Create a PR that updates both `STANDARD.md`_ as well as the `cookiecutter`_
+   templates used by ``reps-new``. These should ideally be done in the same
+   commit.
+3. If necessary, release a new version of ``reps-new``.
+4. If necessary, update existing projects so they conform to the new standard.
```

Comment:
Obviously this is out of scope for this PR, but this paragraph makes me wonder if there's ways we can help automate this? (I have little doubt that projects will easily start to drift in time...)

I'm guessing cookiecutter doesn't provide any help for this.

Reply:
> I'm guessing cookiecutter doesn't provide any help for this.

Yeah not really.

Cookiecutter does allow you to run arbitrary pre/post generation hooks though, so we could set up `reps-new` (or maybe `reps-update?) to work in existing projects. But we'd have to write all the logic that opens and updates all the pre-existing files.